React/src/components/FormControl/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { useForm } from 'react-hook-form';
+
+import { RegisterUser } from 'interfaces/RegisterUser';
+
+import styles from './styles.module.scss';
+
+export function FormControl() {
+  const { register, handleSubmit } = useForm<RegisterUser>();
+
+  return (
+    <div className={styles.pageContainer}>
+      <div className={styles.loginContainer}>
+        <form
+          onSubmit={handleSubmit((data) => console.log({ users: data }))}
+          className={styles.inputsContainer}
+        >
+          <h2 className={styles.dataTitle}>Nombre</h2>
```

Comment:
Remember that inputs should be paired with a `<label>`. It will have accessibility benefits and will help us for interacting with the inputs when we are testing.